### PR TITLE
[SCV-39] Update README

### PR DESCRIPTION
**Pull request by DrewE84**
## Overview
This branch updates the README.md in the root directory of the project.

## Changes
- Updated recommended node.js version to 10.15 (as specified in the .nvmrc)
- Changed `Deployed At` link to reflect current CMR deployment link
- Changed software License.txt

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,13 +1,60 @@
-Copyright 2018-2019 Element 84
+Copyright © 2020 United States Government as represented by the Administrator of the National Aeronautics and Space Administration. All Rights Reserved.
 
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
 
-    http://www.apache.org/licenses/LICENSE-2.0
+    http://www.apache.org/licenses/LICE NSE-2.0
```
Comment (fgabelmannjr) [inline LICENSE.txt:6]: @DrewE84 ,

It looks like we have an errant space in the license URL.

**Comment by fgabelmannjr** [comment]
@DrewE84 , this looks just about ready.  Take a look at the one review item I left for you and we should be good to merge this.

File: LICENSE.txt (hunk 1 above)
Comment (DrewE84) [reply LICENSE.txt]: Good catch! I just pushed the fix.

**Comment by fgabelmannjr** [comment]
This looks good to merge @DrewE84 👍

**Pull request merged**